### ISSUE TITLE
force_apply must have bool or int type

### ISSUE THREAD

**Issue by GadirajuSanjayvarma** [albumentations-team/albumentations#559]
I am new to this albumentations.I used various transformations.When i try to iterate over my dataset i am having error like
force_apply must have bool or int type.
When i try to print the value of force_apply it is of PIL format.
The transformations i apply are
train_transforms = trans.train_transforms([
                                    A.augmentations.transforms.Rotate(limit=(-15.0, 15.0),value=list(fillmeans)),
                                  
                                       A.augmentations.transforms.HorizontalFlip()
                                       ], 
                                   A.augmentations.transforms.Cutout(num_holes=1,fill_value=channel_means)
                                        ])
Plz help me

**Comment by albu** [comment]
pls take a look at the docs. only numpy arrays are supported and you have to pass data into augmentations as named arguments. https://github.com/albumentations-team/albumentations_examples/blob/master/notebooks/showcase.ipynb

**Comment by GadirajuSanjayvarma** [comment]
> pls take a look at the docs. only numpy arrays are supported and you have to pass data into augmentations as named arguments. https://github.com/albumentations-team/albumentations_examples/blob/master/notebooks/showcase.ipynb

I passed arguments as keyword arguments but still i am getting this error.
`import numpy as np
import albumentations as A
import eva4datatransforms
from torchvision import transforms
channel_means = (0.49139968, 0.48215841, 0.44653091)
channel_stdevs = (0.24703223, 0.24348513, 0.26158784)
trans = eva4datatransforms.Transforms(normalize=True, mean=channel_means, stdev=channel_stdevs)
fillmeans = (np.array(channel_means)*255).astype(np.uint8)
# Train Phase transformations
train_transforms = trans.train_transforms([
                                    
                                       A.augmentations.transforms.Rotate(limit=(-15.0,15.0), interpolation=1,
                                        border_mode=4, value=list(fillmeans), mask_value=None, always_apply=False, p=0.5),
                                       #transforms.RandomHorizontalFlip(),
                                       A.augmentations.transforms.HorizontalFlip(always_apply=False, p=0.5),
                                       ], 
                                       
                                       [#transforms.RandomErasing(value = tuple(channel_means)),
                                        A.augmentations.transforms.Cutout(num_holes=8, max_h_size=8, max_w_size=8, fill_value=channel_means, always_apply=False, p=0.5)
                                        ])

# Test Phase transformations
test_transforms = trans.test_transforms()`

Error occured while writing this code
`%matplotlib inline
import matplotlib.pyplot as plt
dataiter=iter(train_loader)
images,labels=next(dataiter)`
